web/src/app/util/ISOPickers.js

```diff
@@ -17,14 +17,16 @@ function hasInputSupport(name) {
 }
 
 function useISOPicker(
-  { value, onChange, timeZone, ...otherProps },
+  { value, onChange, timeZone, min, max, ...otherProps },
   { format, truncateTo, type, Fallback },
 ) {
   const native = hasInputSupport(type)
   const params = useSelector(urlParamSelector)
   const zone = timeZone || params('tz', 'local')
-  const dtValue = DateTime.fromISO(value, { zone })
-  const [inputValue, setInputValue] = useState(dtValue.toFormat(format))
+  const dtValue = value ? DateTime.fromISO(value, { zone }) : null
```

Comment:
It looks like a few changes were mixed in for empty value support, but there's no use case here to actually test/validate in this PR, it would reduce noise to leave it out. I could be missing a use case though